iOS/_Stream Temp/Filter+Extensions.swift

```diff
@@ -13,15 +13,41 @@ extension Filter where Scope: AnyChannelListFilterScope {
 
     /// Filter to match conversations that contain all and only the passed in members.
     static func containOnlyMembers(_ members: [ConversationMember]) -> Filter<Scope> {
+        let userIds = members.map { member in
+            return member.id
+        }
+        return .containsOnly(userIds: userIds)
+    }
```

Comment:
These queries return conversation that only contain the members and no one else. The profile VC does NOT use these.